README.md

```diff
@@ -31,4 +31,73 @@ Po zamknięciu zapisów moderatorzy roku przypisują studentów do grup uwzględ
 - Może przydzielać role
 - Może dodawać, usuwać i edytować konta studentów
 
+
+# Typy encji
+
+## Student
+
+Reprezentuje aktora Student.
+- Imię
+- Nazwisko
+- E-mail
+- Numer indeksu
+
+## Moderator
+
+Podklasa typu encji _Student_
+
+Reprezentuje aktora Moderator.
+
+## Administrator
+
+Podklasa typu encji _Moderator_
+
+Reprezentuje aktora Administrator.
+
+## Rocznik
+
+Reprezentuje grupę użytkowników na tym samym roczniku i kierunku.
+
+- Rok rozpoczęcia nauki
+- Nazwa kierunku
+
+## Zapis
+
+Reprezentuje zapis, w ramach którego studenci będą przypisywani do grup.
+
+- Nazwa zapisu
+- Stan zapisu (otwarty, oczekiwanie na wyniki itp.)
+- Minimalna ilość punktów na przedmiot
+- Maksymalna ilość punktów na przedmiot
+- Maksymalna ilość punktów na grupę
+
+## Przedmiot
+
+Reprezentuje przedmiot, z którego organizowane są zajęcia.
+
+- Nazwa przedmiotu
+- Rodzaj zajęć (wykład, laboratoria itp.)
+- Sposób prowadzenia zajęć (zdalne, stacjonarne itp.)
+
+## Grupa
+
+Reprezentuje grupę zajęciową.
+
+- Prowadzący
```

Comment:
Nie mamy zamodelowanego prowadzącego. Proponuję

## Prowadzący

- Imię
- Nazwisko
- Tytuł naukowy

Reply:
Ew. jakieś sposoby kontaktu też warto wrzucić? Nie wiem jak by to wyglądało wtedy - chyba wszyscy mają na domenie agh.edu.pl

Reply:
Niby można, ale niekoniecznie chcemy dodawać do systemu możliwość kontaktu z prowadzącymi (chyba, że masz jakiś usecase).

Reply:
True, na ten moment nie ma powodu żeby to wprowadzać.